### ISSUE TITLE
Model Zoo

### ISSUE THREAD

**Issue by Idate96** [Pointcept/Pointcept#433]
Any plans to also upload the fine-tuned sonata checkpoints for the indoor/outdoor scenes like has been done with point transformer v3?

**Comment by Gofinge** [comment]
Original experimental records are in the Meta server, and cannot access them and need to re-do these experiments to get these results. So a bit complicated.

Yet, if any model weight would be helpful to your research, let me know which one is needed. I can do the tuning.

**Comment by Idate96** [comment]
Thanks! We are working on RL manipulation (we can use sonata in a student-teacher setup) and navigation projects with our excavator robot (~10 m reach) in group @ RSL ETHZ.
Usually models fine-tuned on outdoor scenes work best, so I'd say sonata finetuned with NuScenes/Waymo would work best (we then do an extra step of fine-tuning on our data for really unstructured terrains).

**Comment by Gofinge** [comment]
In the current version we do indoor SSL and outdoor SSL separately (a bit ugly I know), but we will make them together soon. I will train one version of Sonata with an outdoor dataset again and let you know later. What's the expected feature input for that model? Coordinate and intensity?

**Comment by Idate96** [comment]
**XYZ + Intensity** would probably be a useful baseline for us to start with but having XYZ + RBG seems more general and can be useful more useful for actual manipulation tasks. Our excavator has both LiDAR and camera 360 view. 

Just for contex here's a bit about our current projects:

* **Rock pickup and place:** We use an RL teacher (sim) that gets pointcloud data relative to the rock. To deploy we currently use SAM2 to mask rock points, but we're thinking Sonata could replace this via distillation (student-teacher). For this, XYZ+RGB inputs would be ideal to distinguish rocks from soil, especially since intensity alone isn't always enough. We can project RGB onto the LiDAR points (both in sim and real), and we can use Isaac Lab offline rendering for sim-to-real. 
* **General arm motions (obstacle avoidance):** Tasks like dumping soil into a truck or moving the arm to the next dig location involve pure obstacle avoidance. We have procedurally generated envs in sim where we train the system. We could compute embeddings once per episode, or use a faster raycaster in warp during RL training for XYZ point clouds. For this, XYZ alone is sufficient, but XYZ+Intensity probably is fine too. We could provide a constant intensity layer if needed, but avoiding sensitivity to simulated intensity helps sim-to-real. What would you suggest to provide as intensity if cannot simulate it? 
* **Navigation:** We're looking at potentially swapping parts of our current camera-lidar 3D dynamic mapping system by adding Sonata + decoder for planning. If a fine-tuned XYZ+RGB model is powerful enough, it might even reduce the need for a separate vision model component.

**Comment by Idate96** [comment]
In the past we tried to do semantic segmentation directly on pointcloud and found that pretrained models on indoor dataset don't do too well. But I can try it out! this should be the currently released base model checkpoint right? 
For general arm planning / obstacle avoidance we don't need color though.

**Comment by Gofinge** [comment]
I think tuning the model is necessary, as indoor contents and outdoor contents still have some gap. Yet for outdoor, we usually don’t need such a high resolution of perception, hence set ‘grid_size’ as 0.05m instead of 0.02m. Back to adapting indoor perception model to the outdoor, I think scaling the outdoor point cloud to 2/5 of the original scale and still applying a grid size of “0.02m” should be helpful.

**Comment by Gofinge** [comment]
And if possible, do Sonata pretraining with your own data. Joint training with existing assets would be better.
